### PR TITLE
 [*] parity-codec -> parity-scale-codec 

### PR DESCRIPTION
**Pull request by ascjones**

### CODE REVIEW

File: core/src/env/srml/srml_only/impls.rs [hunk 1]
```diff
@@ -95,9 +95,6 @@ macro_rules! impl_getters_for_srml_env {
             fn $name() -> $ret_type {
                 unsafe { sys::$ext_name() };
                 Decode::decode(&mut &read_scratch_buffer()[..])
-                    .ok_or(concat!(
-                        stringify!($name), " received an incorrectly sized buffer from SRML"
-                    ))
                     .expect(concat!(
                         stringify!($name), " expects to receive a correctly sized buffer"
                     ))
```
Comment (Robbepop) [inline core/src/env/srml/srml_only/impls.rs]: Hmm strange what was the semantics before your change? :confused: Good you fixed that!

File: core/src/storage/cell/typed_cell.rs [hunk 2]
```diff
@@ -79,19 +79,22 @@ impl<T> TypedCell<T> {
...
 where
-    T: parity_codec::Decode,
+    T: scale::Decode,
 {
     /// Loads the value stored in the cell if any.
     pub fn load(&self) -> Option<T> {
-        self.cell
-            .load()
-            .and_then(|bytes| T::decode(&mut &bytes[..]))
+        self.cell.load().map(|bytes| {
```
[the remark this answers is not in the document]
Comment (ascjones) [reply core/src/storage/cell/typed_cell.rs]: And now matches semantics at https://github.com/paritytech/ink/blob/master/core/src/storage/chunk/typed_chunk.rs#L150

File: core/src/env/srml/srml_only/impls.rs (hunk 1 above)
Comment (ascjones) [reply core/src/env/srml/srml_only/impls.rs]: Yeah it just would have meant the first error message was erased. Looks like something that could have happened in a merge conflict resolution.

File: core/src/storage/cell/typed_cell.rs (hunk 2 above)
[the remark this answers is not in the document]
Comment (Robbepop) [reply core/src/storage/cell/typed_cell.rs]: I still like the comment there. Good stuff for another simple PR.

**Pull request merged**